EventFilter/Utilities/src/FedRawDataInputSource.cc

```diff
@@ -398,6 +398,8 @@ inline evf::EvFDaqDirector::FileStatus FedRawDataInputSource::getNextEvent() {
         maybeOpenNewLumiSection(currentFile_->lumi_);
       }
     //immediately delete empty file
+    if (currentFile_->rawFd_ != -1)
+      close(currentFile_->rawFd_);
```

Comment:
I will change it. A small caveat is, with current code it would mean deleting file before closing it, so that has to be rearranged. File deletion is in some cases done later (to make sure that file is present until after events are processed) so it's not done in InputFile class.

Reply:
After another look, the object is always removed when the file is, so it can be done in destructor.